### PR TITLE
Compact no more

### LINKED ISSUE #425: [BUG] documentation on exist-db.org is screwed up
See - http://www.exist-db.org/exist/apps/doc/configuration

Screenshot 2020-01-21 at 21 40 19

1. Note the right-side menu "Contents" is just `element` over and over
2. Sections seem to be missing
3. Term list of "divided into eleven sections" is mangled up.

### PR DESCRIPTION
**Pull request by duncdrum**
Removes the ever troublesome compact variableList. Minor fixes in various articles. 

ToC generation now works with inline elements, and has a test.

Close #425

### CODE REVIEW

File: src/main/xar-resources/data/ant-tasks/ant-tasks.xml [hunk 1]
```diff
@@ -250,17 +250,17 @@
         <programlisting language="xml" xlink:href="listings/listing-8.xml"/>
         <programlisting language="xml" xlink:href="listings/listing-9.xml"/>
         <para>Attributes:</para>
-        <variablelist spacing="compact">
+        <variablelist>
             <varlistentry>
                 <term><code>resources</code></term>
                 <listitem>
-                    <para>If <code>true</code> lists resources</para>
+                    <para>If <code>true</code> return a list of resources.</para>
```
Comment (lguariento) [inline src/main/xar-resources/data/ant-tasks/ant-tasks.xml]: Better id 'return**s**'

File: src/main/xar-resources/data/ant-tasks/ant-tasks.xml [hunk 2]
```diff
@@ -250,17 +250,17 @@
...
                 </listitem>
             </varlistentry>
             <varlistentry>
                 <term><code>collections</code></term>
                 <listitem>
-                    <para>If <code>true</code> lists collections</para>
+                    <para>If <code>true</code> return a list of collections.</para>
```
Comment (lguariento) [inline src/main/xar-resources/data/ant-tasks/ant-tasks.xml]: _idem_

File: src/main/xar-resources/data/ant-tasks/ant-tasks.xml [hunk 3]
```diff
@@ -728,11 +727,11 @@
         <para>The <code>addgroup</code> task adds a user group. </para>
         <programlisting language="xml" xlink:href="listings/listing-27.xml"/>
         <para>Attributes:</para>
-        <variablelist spacing="compact">
+        <variablelist>
             <varlistentry>
                 <term><code>name</code></term>
                 <listitem>
-                    <para>Name of the new group.</para>
+                    <para>Takes thename of the new group.</para>
```
Comment (lguariento) [inline src/main/xar-resources/data/ant-tasks/ant-tasks.xml]: missing space between "the" and "name".

Also, either 'groupname' here and in the following <para>, or name in both cases. I see that you use 'username of the user' later on, so perhaps we want to keep groupname everywhere.

File: src/main/xar-resources/data/ant-tasks/ant-tasks.xml [hunk 4]
```diff
@@ -746,11 +745,11 @@
         <para>The <code>rmgroup</code> task removes a user group.</para>
         <programlisting language="xml" xlink:href="listings/listing-28.xml"/>
         <para>Attributes:</para>
-        <variablelist spacing="compact">
+        <variablelist>
             <varlistentry>
                 <term><code>name</code></term>
                 <listitem>
-                    <para>Name of the group to remove.</para>
+                    <para>Takes the groupname of the group to remove.</para>
```
Comment (lguariento) [inline src/main/xar-resources/data/ant-tasks/ant-tasks.xml]: See previous note about consistency (groupname of the group/name of the group)

**Comment by duncdrum** [comment]
hold on a second

File: src/main/xar-resources/data/jmx/jmx.xml [hunk 5]
```diff
@@ -72,43 +72,43 @@
...
                     <listitem>
-                        <para>displays server statistics on cache and memory usage.</para>
+                        <para>Displays server statistics on cache and memory usage.</para>
                     </listitem>
                 </varlistentry>
                 <varlistentry>
                     <term><code>-d, --db</code></term>
                     <listitem>
-                        <para>display general info about the db instance.</para>
+                        <para>Display general info about the db instance.</para>
```
Comment (lguariento) [inline src/main/xar-resources/data/jmx/jmx.xml]: Display**s**

File: src/main/xar-resources/data/jmx/jmx.xml [hunk 6]
```diff
@@ -72,43 +72,43 @@
...
                     </listitem>
                 </varlistentry>
                 <varlistentry>
                     <term><code>-h, --help</code></term>
                     <listitem>
-                        <para>print help on command line options and exit.</para>
+                        <para>Print help on command line options and exit.</para>
```
Comment (lguariento) [inline src/main/xar-resources/data/jmx/jmx.xml]: Print**s**

File: src/main/xar-resources/data/jmx/jmx.xml [hunk 7]
```diff
@@ -123,27 +123,25 @@
...
+                        <para>Retrieves the most recent sanity/consistency check report.</para>
                     </listitem>
                 </varlistentry>
                 <varlistentry>
-                    <term><code>-w, --wait &lt;argument&gt;</code></term>
+                    <term><code>-w, --wait</code> <tag>argument</tag></term>
                     <listitem>
-                        <para> while displaying server statistics: keep retrieving statistics, but
-                            wait the specified number of seconds between calls.</para>
+                        <para> while displaying server statistics: keep retrieving statistics, but wait the specified number of seconds between calls.</para>
```
Comment (lguariento) [inline src/main/xar-resources/data/jmx/jmx.xml]: keeps, waits

**Pull request merged**